framework/qlf/dashboard/bokeh/utils/scalar_metrics.py

```diff
@@ -153,3 +153,65 @@ def Load_metrics_n_tests(self):
                 dic_met.update({i: aux['metrics']})
                 dic_tst.update({i: aux['params']})
         return dic_met, dic_tst
+
+    def step_color(self, step_name):
```

Comment:
Is this method still used? I think we moved it to https://github.com/desihub/qlf/blob/master/framework/bin/scalar_metrics.py